src/store/flow/views/import.ts

```diff
@@ -249,18 +249,6 @@ export const actions: ActionTree<IImportState, IRootState> = {
         isValid = false
       }
 
-      if (flowContainer.flows?.length > 1) {
```

Comment:
let's decline the other PR https://github.com/FLOIP/flow-builder/pull/447 in favor of this one then. 
cc: @bulatgab as you already approved it